CHANGELOG.md

```diff
@@ -1,7 +1,12 @@
-# dbt_facebook_ads_source version.version
+# dbt_facebook_ads_source v0.8.1
+
+## Bug Fixes
+- Added logic to the `stg_facebook_ads__creative_history` model to dynamically convert the `url_tags` field from a JSON to a STRING if necessary (BigQuery only). This is necessary for the success of downstream transformations in the `facebook_ads` package. ([PR #43](https://github.com/fivetran/dbt_facebook_ads_source/pull/43))
+> Note: If you are a BigQuery user unioning multiple Facebook Ads connections together in the package, the data type of `creative_history.url_tags` must be consistent across your connections. Otherwise, the `stg_facebook_ads__creative_history_tmp` will likely fail. Please reach out and create an issue if you are facing this error.
 
 ## Documentation
 - Corrected references to connectors and connections in the README. ([#40](https://github.com/fivetran/dbt_facebook_ads_source/pull/40))
+- Adjusted the header formatting in the README. ([PR #43](https://github.com/fivetran/dbt_facebook_ads_source/pull/43))
```

Comment:
Very tiny, but you can add the License update here too

Reply:
Ah yes!

Reply:
Just added